### DISCUSSION TITLE
External library is failing to load with "__webpack_require__ is not defined" error

### DISCUSSION THREAD

**Discussion by iamravisingh** [web-infra-dev/rsbuild#2849] [Q&A]
Hello Folks, 

One of my library `@here/maps-api-for-javascript` use to render a map is falling with below error,
`__webpack_require__ is not defined`.

In our existing CRA project we use to handle this issue with below configuration 
https://stackoverflow.com/questions/66159523/webpack-require-is-not-defined-here-map-bundle-doesnt-work

As other libraries do not have such issue, I am assuming it's something related to the library I am using. 
Is there a way to handle such a scenario in `rsbuild.config.ts`? 

<img width="733" alt="image" src="https://github.com/web-infra-dev/rsbuild/assets/34270887/6ef6af96-c609-45e2-85c0-d4769cf34c9b">

**Comment by SoonIter** [comment]
Rsbuild allows users to directly modify the Rspack configuration transparently
https://www.rspack.dev/config/node#nodeglobal
[`tools.rspack`](https://rsbuild.dev/config/tools/rspack)

And you can debug the configuration of Rspack through `DEBUG=rspack rsbuild build`
